test/pg-cdc/pg-cdc.td

```diff
@@ -266,39 +266,47 @@ incorrect column specification: 3 columns were specified, upstream has 2: pk, f2
+  PUBLICATION 'mz_source' TABLE pk_table (pk INTEGER PRIMARY KEY, f2 TEXT);
 
 > CREATE MATERIALIZED SOURCE "nonpk_table"
   FROM POSTGRES HOST 'host=postgres port=5432 user=postgres password=postgres dbname=postgres'
-  PUBLICATION 'mz_source' NAMESPACE 'public' TABLE 'nonpk_table' (f1 INTEGER, f2 INTEGER);
+  PUBLICATION 'mz_source' TABLE nonpk_table (f1 INTEGER, f2 INTEGER);
 
 > CREATE MATERIALIZED SOURCE "types_table"
   FROM POSTGRES HOST 'host=postgres port=5432 user=postgres password=postgres dbname=postgres'
-  PUBLICATION 'mz_source' NAMESPACE 'public' TABLE 'types_table' (date_col DATE, time_col TIME, timestamp_col TIMESTAMP, uuid_col UUID, double_col DOUBLE PRECISION);
+  PUBLICATION 'mz_source' TABLE types_table (date_col DATE, time_col TIME, timestamp_col TIMESTAMP, uuid_col UUID, double_col DOUBLE PRECISION);
 
 > CREATE MATERIALIZED SOURCE "large_text"
   FROM POSTGRES HOST 'host=postgres port=5432 user=postgres password=postgres dbname=postgres'
-  PUBLICATION 'mz_source' NAMESPACE 'public' TABLE 'large_text' (f1 TEXT);
+  PUBLICATION 'mz_source' TABLE large_text (f1 TEXT);
 
 > CREATE MATERIALIZED SOURCE "trailing_space_pk"
   FROM POSTGRES HOST 'host=postgres port=5432 user=postgres password=postgres dbname=postgres'
-  PUBLICATION 'mz_source' NAMESPACE 'public' TABLE 'trailing_space_pk' (f1 TEXT PRIMARY KEY);
+  PUBLICATION 'mz_source' TABLE trailing_space_pk (f1 TEXT PRIMARY KEY);
 
 > CREATE MATERIALIZED SOURCE "trailing_space_nopk"
   FROM POSTGRES HOST 'host=postgres port=5432 user=postgres password=postgres dbname=postgres'
-  PUBLICATION 'mz_source' NAMESPACE 'public' TABLE 'trailing_space_nopk' (f1 TEXT);
+  PUBLICATION 'mz_source' TABLE trailing_space_nopk (f1 TEXT);
 
 > CREATE MATERIALIZED SOURCE "multipart_pk"
   FROM POSTGRES HOST 'host=postgres port=5432 user=postgres password=postgres dbname=postgres'
-  PUBLICATION 'mz_source' NAMESPACE 'public' TABLE 'multipart_pk' (f1 INTEGER PRIMARY KEY, f2 TEXT PRIMARY KEY NOT NULL, f3 TEXT);
+  PUBLICATION 'mz_source' TABLE multipart_pk (f1 INTEGER PRIMARY KEY, f2 TEXT PRIMARY KEY NOT NULL, f3 TEXT);
 
 > CREATE MATERIALIZED SOURCE "nulls_table"
   FROM POSTGRES HOST 'host=postgres port=5432 user=postgres password=postgres dbname=postgres'
-  PUBLICATION 'mz_source' NAMESPACE 'public' TABLE 'nulls_table' (f1 TEXT, f2 INTEGER);
+  PUBLICATION 'mz_source' TABLE nulls_table (f1 TEXT, f2 INTEGER);
 
 > CREATE MATERIALIZED SOURCE "utf8_table"
   FROM POSTGRES HOST 'host=postgres port=5432 user=postgres password=postgres dbname=postgres'
-  PUBLICATION 'mz_source' NAMESPACE 'public' TABLE 'utf8_table' (f1 TEXT PRIMARY KEY, f2 TEXT);
+  PUBLICATION 'mz_source' TABLE utf8_table (f1 TEXT PRIMARY KEY, f2 TEXT);
+
+> CREATE MATERIALIZED SOURCE "таблица"
+  FROM POSTGRES HOST 'host=postgres port=5432 user=postgres password=postgres dbname=postgres'
+  PUBLICATION 'mz_source' TABLE "таблица" ("колона" TEXT);
+
+> CREATE MATERIALIZED SOURCE "conflict_table"
```

Comment:
It might be worthwhile to give `conflict_table` and `conflict_schema.conflict_table` conflicting column types as well, in order to check our column validation code. So something like:
```
CREATE TABLE conflict_table (f1 INTEGER);
...

CREATE SCHEMA conflict_schema;
CREATE TABLE conflict_schema.conflict_table (f1 TEXT);
...
INSERT INTO conflict_schema.conflict_table VALUES ("234");

! CREATE MATERIALIZED SOURCE "conflict_table"
  FROM POSTGRES HOST 'host=postgres port=5432 user=postgres password=postgres dbname=postgres'
  PUBLICATION 'mz_source' TABLE conflict_table (f1 TEXT)
(error that the field is an int)
```
But this is a small thought!

Reply:
Added.